rpcs3/Emu/VFS.cpp

```diff
@@ -68,7 +68,19 @@ bool vfs::mount(std::string_view vpath, std::string_view path)
 		if (pos == umax)
 		{
 			// Mounting completed
-			list.back()->path = path;
+			auto& dst = list.back()->path;
+			dst = fs::resolve_path(path);
+
+			if (dst.empty())
+			{
+				return false;
+			}
+
+			if (!dst.ends_with(fs::delim))
```

Comment:
Path wasn't supposed to be changed inside vfs::mount
Especially not like this, it's not enforced to be limited to directories only

Reply:
Now delimiter suffixing trait is unchanged.